Delcam.ProductInterface.PowerMILL.Test/PMEntityTests/PMProjectTests.cs

```diff
@@ -125,6 +125,30 @@ public void ActiveWorldWorkplaneTest()
             Assert.That(_powerMILL.ActiveProject.Workplanes.All(x => x.IsActive == false), Is.True);
         }
 
+        [Test]
+        public void CreateBlockFromBoundary()
+        {   
+            _powerMILL.Execute(string.Format("CREATE BOUNDARY \"{0}\"", "TestBoundary"));            
+            _powerMILL.Execute(string.Format("EDIT BOUNDARY \"{0}\" INSERT FILE \"{1}\"", "TestBoundary", TestFiles.CurvesFiles));
+            _powerMILL.ActiveProject.Refresh();
+            PMBoundary boundary = _powerMILL.ActiveProject.Boundaries.ActiveItem;
+            _powerMILL.ActiveProject.CreateBlock(boundary, 0, 100);
+            Assert.AreEqual("100,0", _powerMILL.ExecuteEx("PRINT $BLOCK.LIMITS.ZMax"));
```

Comment:
Can we add properties for the block limits while we are here?  Perhaps we can get a BoundingBox back?

Reply:
Do you mean something like this?
`Autodesk.Geometry.BoundingBox boundingBox = _powerMILL.ActiveProject.CreateBlockFromBoundaryWithLimits(boundary, 0, 100);
Assert.AreEqual(100, boundingBox.MaxZ);`

Reply:
Yes, looks great.  An even better way to do the assert is:
Assert.That(boundingBox.MaxZ, Is.EqualTo(100));